ui/__config__/webpack.config.js

```diff
@@ -44,8 +44,17 @@ const config = {
             },
             {
                 test: /\.(js|jsx)$/,
-                exclude: /(node_modules)/,
-                loader: 'babel-loader',
+                oneOf: [
+                    {
+                        test: /(\/__inline__)/,
+                        exclude: /node_modules/,
+                        use: ['raw-loader', { loader: 'babel-loader' }],
```

Comment:
Since `babel-loader` doesn't need any additional options, I think we can shorted this to:
```js
use: ['raw-loader', 'babel-loader']
```
Same on line 55